pkg/reconciler/dispatcher/jetstream/nats_jetstreaming_schannel.go

```diff
@@ -82,10 +89,16 @@ type envConfig struct {
 
 // NewController initializes the controller and is called by the generated code.
 // Registers event handlers to enqueue events.
-func NewController(ctx context.Context, _ configmap.Watcher) *controller.Impl {
+func NewController(ctx context.Context, watcher configmap.Watcher) *controller.Impl {
 
 	logger := logging.FromContext(ctx)
 
+	// Setup trace publishing.
+	iw := watcher.(*configmapinformer.InformedWatcher)
+	if err := setupDynamicPublishing(logger, iw, controllerAgentName, tracingconfig.ConfigName); err != nil {
+		logger.Panicw("Error setting up trace publishing", zap.Error(err))
```

Comment:
*nit*: It looks like we use `logger.Fatalw()` just below; maybe use that for consistency?